src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs

```diff
@@ -27,11 +27,36 @@ public class GenevaExporterOptions
         [Schema.V40.PartA.Ver] = "4.0",
     };
 
+    private IReadOnlyDictionary<string, string> _tableNameMappings;
+
     public string ConnectionString { get; set; }
 
     public IEnumerable<string> CustomFields { get; set; }
 
-    public IReadOnlyDictionary<string, string> TableNameMappings { get; set; }
+    public IReadOnlyDictionary<string, string> TableNameMappings
+    {
+        get => this._tableNameMappings;
+        set
+        {
+            Guard.ThrowIfNull(value);
+
+            foreach (var entry in value)
+            {
+                if (entry.Value is null)
```

Comment:
we can check if key is null/empty as well here.

Reply:
Actually, C# already doesn't allow `null` to be a key in a Dictionary :)